*** Starting uWSGI 2.0.19.1 (64bit) on [Fri Apr  9 22:50:03 2021] ***
compiled with version: 8.3.1 20191121 (Red Hat 8.3.1-5) on 09 April 2021 13:31:58
os: Linux-4.18.0-240.10.1.el8_3.x86_64 #1 SMP Mon Jan 18 17:05:51 UTC 2021
nodename: buaa-2033
machine: x86_64
clock source: unix
detected number of CPU cores: 2
current working directory: /root/ReedSailing-BackEnd/django_backend
detected binary path: /root/env/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to ./
your processes number limit is 30951
your memory page size is 4096 bytes
detected max file descriptor number: 65535
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address :8000 fd 6
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.8.1 (default, Apr  9 2021, 01:45:33)  [GCC 8.3.1 20191121 (Red Hat 8.3.1-5)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0xeab410
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 802120 bytes (783 KB) for 10 cores
*** Operational MODE: preforking ***
added /root/env/lib/python3.8/site-packages/ to pythonpath.
ModuleNotFoundError: No module named 'buaa'
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 259699)
spawned uWSGI worker 1 (pid: 259700, cores: 1)
spawned uWSGI worker 2 (pid: 259701, cores: 1)
spawned uWSGI worker 3 (pid: 259702, cores: 1)
spawned uWSGI worker 4 (pid: 259703, cores: 1)
spawned uWSGI worker 5 (pid: 259704, cores: 1)
spawned uWSGI worker 6 (pid: 259705, cores: 1)
spawned uWSGI worker 7 (pid: 259706, cores: 1)
spawned uWSGI worker 8 (pid: 259707, cores: 1)
spawned uWSGI worker 9 (pid: 259708, cores: 1)
spawned uWSGI worker 10 (pid: 259709, cores: 1)
unable to stat() ./reload, events will be triggered as soon as the file is created
Fri Apr  9 22:50:13 2021 - *** ./reload has been touched... grace them all !!! ***
...gracefully killing workers...
Gracefully killing worker 1 (pid: 259700)...
Gracefully killing worker 2 (pid: 259701)...
Gracefully killing worker 5 (pid: 259704)...
Gracefully killing worker 3 (pid: 259702)...
Gracefully killing worker 6 (pid: 259705)...
Gracefully killing worker 4 (pid: 259703)...
Gracefully killing worker 7 (pid: 259706)...
Gracefully killing worker 8 (pid: 259707)...
Gracefully killing worker 10 (pid: 259709)...
Gracefully killing worker 9 (pid: 259708)...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
worker 5 buried after 1 seconds
worker 6 buried after 1 seconds
worker 7 buried after 1 seconds
worker 8 buried after 1 seconds
worker 9 buried after 1 seconds
worker 10 buried after 1 seconds
binary reloading uWSGI...
chdir() to /root/ReedSailing-BackEnd/django_backend
closing all non-uwsgi socket fds > 2 (max_fd = 65535)...
found fd 6 mapped to socket 0 (:8000)
running /root/env/bin/uwsgi
[uWSGI] getting INI configuration from uwsgi.ini
*** Starting uWSGI 2.0.19.1 (64bit) on [Fri Apr  9 22:50:15 2021] ***
compiled with version: 8.3.1 20191121 (Red Hat 8.3.1-5) on 09 April 2021 13:31:58
os: Linux-4.18.0-240.10.1.el8_3.x86_64 #1 SMP Mon Jan 18 17:05:51 UTC 2021
nodename: buaa-2033
machine: x86_64
clock source: unix
detected number of CPU cores: 2
current working directory: /root/ReedSailing-BackEnd/django_backend
detected binary path: /root/env/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to ./
your processes number limit is 30951
your memory page size is 4096 bytes
detected max file descriptor number: 65535
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 inherited INET address :8000 fd 6
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.8.1 (default, Apr  9 2021, 01:45:33)  [GCC 8.3.1 20191121 (Red Hat 8.3.1-5)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x152e2f0
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 802120 bytes (783 KB) for 10 cores
*** Operational MODE: preforking ***
added /root/env/lib/python3.8/site-packages/ to pythonpath.
ModuleNotFoundError: No module named 'buaa'
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
gracefully (RE)spawned uWSGI master process (pid: 259699)
spawned uWSGI worker 1 (pid: 259714, cores: 1)
spawned uWSGI worker 2 (pid: 259715, cores: 1)
spawned uWSGI worker 3 (pid: 259716, cores: 1)
spawned uWSGI worker 4 (pid: 259717, cores: 1)
spawned uWSGI worker 5 (pid: 259718, cores: 1)
spawned uWSGI worker 6 (pid: 259719, cores: 1)
spawned uWSGI worker 7 (pid: 259720, cores: 1)
spawned uWSGI worker 8 (pid: 259721, cores: 1)
spawned uWSGI worker 9 (pid: 259722, cores: 1)
spawned uWSGI worker 10 (pid: 259723, cores: 1)
--- no python application found, check your startup logs for errors ---
[pid: 259722|app: -1|req: -1/1] 121.36.65.184 () {34 vars in 509 bytes} [Sat Apr 10 01:31:42 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21327 (max 4096)...skip
-- unavailable modifier requested: 3 --
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 16709 (max 4096)...skip
-- unavailable modifier requested: 4 --
uwsgi_proto_uwsgi_parser(): Resource temporarily unavailable [proto/uwsgi.c line 40]
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21327 (max 4096)...skip
invalid request block size: 16709 (max 4096)...skip
invalid request block size: 16709 (max 4096)...skip
-- unavailable modifier requested: 22 --
-- unavailable modifier requested: 22 --
uwsgi_proto_uwsgi_parser(): Resource temporarily unavailable [proto/uwsgi.c line 40]
uwsgi_proto_uwsgi_parser(): Resource temporarily unavailable [proto/uwsgi.c line 40]
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21327 (max 4096)...skip
invalid request block size: 21327 (max 4096)...skip
--- no python application found, check your startup logs for errors ---
[pid: 259722|app: -1|req: -1/6] 42.236.10.78 () {44 vars in 930 bytes} [Sat Apr 10 19:58:22 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
-- unavailable modifier requested: 4 --
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
-- unavailable modifier requested: 3 --
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21327 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
-- unavailable modifier requested: 5 --
-- unavailable modifier requested: 4 --
invalid request block size: 21584 (max 4096)...skip
uwsgi_proto_uwsgi_parser(): Resource temporarily unavailable [proto/uwsgi.c line 40]
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 16709 (max 4096)...skip
invalid request block size: 16709 (max 4096)...skip
invalid request block size: 16709 (max 4096)...skip
--- no python application found, check your startup logs for errors ---
[pid: 259721|app: -1|req: -1/11] 51.15.209.146 () {38 vars in 589 bytes} [Sun Apr 11 08:53:07 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 16709 (max 4096)...skip
-- unavailable modifier requested: 3 --
invalid request block size: 16709 (max 4096)...skip
invalid request block size: 16709 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
-- unavailable modifier requested: 22 --
-- unavailable modifier requested: 22 --
uwsgi_proto_uwsgi_parser(): Resource temporarily unavailable [proto/uwsgi.c line 40]
uwsgi_proto_uwsgi_parser(): Resource temporarily unavailable [proto/uwsgi.c line 40]
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 16709 (max 4096)...skip
invalid request block size: 21327 (max 4096)...skip
invalid request block size: 16709 (max 4096)...skip
invalid request block size: 16709 (max 4096)...skip
-- unavailable modifier requested: 3 --
-- unavailable modifier requested: 3 --
--- no python application found, check your startup logs for errors ---
[pid: 259722|app: -1|req: -1/17] 124.70.56.97 () {34 vars in 528 bytes} [Mon Apr 12 23:27:31 2021] GET /robots.txt => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
invalid request block size: 21327 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
-- unavailable modifier requested: 4 --
uwsgi_proto_uwsgi_parser(): Resource temporarily unavailable [proto/uwsgi.c line 40]
invalid request block size: 21573 (max 4096)...skip
-- unavailable modifier requested: 4 --
invalid request block size: 21573 (max 4096)...skip
uwsgi_proto_uwsgi_parser(): Resource temporarily unavailable [proto/uwsgi.c line 40]
-- unavailable modifier requested: 22 --
-- unavailable modifier requested: 22 --
uwsgi_proto_uwsgi_parser(): Resource temporarily unavailable [proto/uwsgi.c line 40]
uwsgi_proto_uwsgi_parser(): Resource temporarily unavailable [proto/uwsgi.c line 40]
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21327 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 16709 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 16709 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 29285 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 17747 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21327 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
-- unavailable modifier requested: 4 --
uwsgi_proto_uwsgi_parser(): Resource temporarily unavailable [proto/uwsgi.c line 40]
invalid request block size: 21327 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 16709 (max 4096)...skip
invalid request block size: 16709 (max 4096)...skip
-- unavailable modifier requested: 22 --
-- unavailable modifier requested: 22 --
uwsgi_proto_uwsgi_parser(): Resource temporarily unavailable [proto/uwsgi.c line 40]
uwsgi_proto_uwsgi_parser(): Resource temporarily unavailable [proto/uwsgi.c line 40]
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21327 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
-- unavailable modifier requested: 4 --
invalid request block size: 21573 (max 4096)...skip
-- unavailable modifier requested: 3 --
invalid request block size: 21327 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 16709 (max 4096)...skip
--- no python application found, check your startup logs for errors ---
[pid: 259722|app: -1|req: -1/27] 47.112.193.218 () {36 vars in 445 bytes} [Thu Apr 15 16:47:41 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
-- unavailable modifier requested: 22 --
-- unavailable modifier requested: 22 --
uwsgi_proto_uwsgi_parser(): Resource temporarily unavailable [proto/uwsgi.c line 40]
uwsgi_proto_uwsgi_parser(): Resource temporarily unavailable [proto/uwsgi.c line 40]
invalid request block size: 21573 (max 4096)...skip
--- no python application found, check your startup logs for errors ---
[pid: 259722|app: -1|req: -1/30] 47.112.193.218 () {36 vars in 445 bytes} [Thu Apr 15 21:17:37 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 16709 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21327 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
--- no python application found, check your startup logs for errors ---
[pid: 259722|app: -1|req: -1/31] 47.112.193.218 () {36 vars in 445 bytes} [Fri Apr 16 10:16:27 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
--- no python application found, check your startup logs for errors ---
[pid: 259722|app: -1|req: -1/32] 47.112.193.218 () {36 vars in 445 bytes} [Fri Apr 16 13:13:01 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 259722|app: -1|req: -1/33] 47.112.193.218 () {36 vars in 445 bytes} [Fri Apr 16 13:45:04 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 259722|app: -1|req: -1/34] 47.112.193.218 () {36 vars in 445 bytes} [Fri Apr 16 13:45:06 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 18515 (max 4096)...skip
-- unavailable modifier requested: 3 --
invalid request block size: 21327 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
--- no python application found, check your startup logs for errors ---
[pid: 259722|app: -1|req: -1/36] 117.78.6.19 () {34 vars in 507 bytes} [Sat Apr 17 02:35:00 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
-- unavailable modifier requested: 22 --
-- unavailable modifier requested: 22 --
uwsgi_proto_uwsgi_parser(): Resource temporarily unavailable [proto/uwsgi.c line 40]
uwsgi_proto_uwsgi_parser(): Resource temporarily unavailable [proto/uwsgi.c line 40]
invalid request block size: 21573 (max 4096)...skip
--- no python application found, check your startup logs for errors ---
[pid: 259722|app: -1|req: -1/39] 47.112.193.218 () {36 vars in 445 bytes} [Sat Apr 17 08:13:01 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 259722|app: -1|req: -1/40] 47.112.193.218 () {36 vars in 445 bytes} [Sat Apr 17 09:34:13 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 259722|app: -1|req: -1/41] 47.112.193.218 () {36 vars in 445 bytes} [Sat Apr 17 10:33:41 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 259722|app: -1|req: -1/42] 47.112.193.218 () {36 vars in 445 bytes} [Sat Apr 17 11:43:44 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 259722|app: -1|req: -1/43] 47.112.193.218 () {36 vars in 445 bytes} [Sat Apr 17 12:30:59 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
invalid request block size: 21573 (max 4096)...skip
--- no python application found, check your startup logs for errors ---
[pid: 259721|app: -1|req: -1/44] 47.112.193.218 () {36 vars in 445 bytes} [Sat Apr 17 14:13:40 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 259722|app: -1|req: -1/45] 47.112.193.218 () {36 vars in 445 bytes} [Sat Apr 17 14:52:59 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 259722|app: -1|req: -1/46] 47.112.193.218 () {36 vars in 445 bytes} [Sat Apr 17 16:33:06 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 259722|app: -1|req: -1/47] 47.112.193.218 () {36 vars in 445 bytes} [Sat Apr 17 20:43:51 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 259722|app: -1|req: -1/48] 47.112.193.218 () {36 vars in 445 bytes} [Sat Apr 17 21:13:36 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
-- unavailable modifier requested: 3 --
--- no python application found, check your startup logs for errors ---
[pid: 259722|app: -1|req: -1/50] 47.112.193.218 () {36 vars in 445 bytes} [Sat Apr 17 22:43:29 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
-- unavailable modifier requested: 4 --
invalid request block size: 21573 (max 4096)...skip
--- no python application found, check your startup logs for errors ---
[pid: 259722|app: -1|req: -1/52] 47.112.193.218 () {36 vars in 445 bytes} [Sat Apr 17 23:43:28 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
-- unavailable modifier requested: 3 --
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 16709 (max 4096)...skip
invalid request block size: 16709 (max 4096)...skip
invalid request block size: 16709 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
-- unavailable modifier requested: 22 --
-- unavailable modifier requested: 22 --
uwsgi_proto_uwsgi_parser(): Resource temporarily unavailable [proto/uwsgi.c line 40]
uwsgi_proto_uwsgi_parser(): Resource temporarily unavailable [proto/uwsgi.c line 40]
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 16709 (max 4096)...skip
--- no python application found, check your startup logs for errors ---
[pid: 259723|app: -1|req: -1/56] 47.112.193.218 () {36 vars in 445 bytes} [Sun Apr 18 10:13:45 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 16709 (max 4096)...skip
invalid request block size: 16709 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
--- no python application found, check your startup logs for errors ---
[pid: 259722|app: -1|req: -1/57] 47.112.193.218 () {36 vars in 445 bytes} [Sun Apr 18 21:13:57 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
invalid request block size: 16709 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 16709 (max 4096)...skip
-- unavailable modifier requested: 3 --
invalid request block size: 44991 (max 4096)...skip
invalid request block size: 21327 (max 4096)...skip
-- unavailable modifier requested: 23 --
-- unavailable modifier requested: 23 --
invalid request block size: 53333 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21327 (max 4096)...skip
--- no python application found, check your startup logs for errors ---
[pid: 259717|app: -1|req: -1/61] 67.242.172.238 () {38 vars in 695 bytes} [Mon Apr 19 11:53:01 2021] GET /wordpress/wp-includes/wlwmanifest.xml => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
-- unavailable modifier requested: 22 --
-- unavailable modifier requested: 22 --
uwsgi_proto_uwsgi_parser(): Resource temporarily unavailable [proto/uwsgi.c line 40]
uwsgi_proto_uwsgi_parser(): Resource temporarily unavailable [proto/uwsgi.c line 40]
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 16709 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
--- no python application found, check your startup logs for errors ---
[pid: 259722|app: -1|req: -1/64] 47.112.193.218 () {36 vars in 445 bytes} [Mon Apr 19 19:40:40 2021] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
invalid request block size: 58881 (max 4096)...skip
*** Starting uWSGI 2.0.19.1 (64bit) on [Tue Apr 20 01:29:54 2021] ***
compiled with version: 8.3.1 20191121 (Red Hat 8.3.1-5) on 08 April 2021 18:25:32
os: Linux-4.18.0-240.10.1.el8_3.x86_64 #1 SMP Mon Jan 18 17:05:51 UTC 2021
nodename: buaa-2033
machine: x86_64
clock source: unix
detected number of CPU cores: 2
current working directory: /root/ReedSailing-BackEnd/django_backend
detected binary path: /root/python/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to ./
your processes number limit is 30951
your memory page size is 4096 bytes
detected max file descriptor number: 65535
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.8.1 (default, Apr  9 2021, 01:45:33)  [GCC 8.3.1 20191121 (Red Hat 8.3.1-5)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x2126330
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
The -s/--socket option is missing and stdin is not a socket.
Tue Apr 20 01:30:01 2021 - *** ./reload has been touched... grace them all !!! ***
...gracefully killing workers...
Gracefully killing worker 1 (pid: 259714)...
Gracefully killing worker 3 (pid: 259716)...
Gracefully killing worker 2 (pid: 259715)...
Gracefully killing worker 4 (pid: 259717)...
Gracefully killing worker 6 (pid: 259719)...
Gracefully killing worker 5 (pid: 259718)...
Gracefully killing worker 8 (pid: 259721)...
Gracefully killing worker 7 (pid: 259720)...
Gracefully killing worker 9 (pid: 259722)...
Gracefully killing worker 10 (pid: 259723)...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
worker 5 buried after 1 seconds
worker 6 buried after 1 seconds
worker 7 buried after 1 seconds
worker 8 buried after 1 seconds
worker 9 buried after 1 seconds
worker 10 buried after 1 seconds
binary reloading uWSGI...
chdir() to /root/ReedSailing-BackEnd/django_backend
closing all non-uwsgi socket fds > 2 (max_fd = 65535)...
found fd 6 mapped to socket 0 (:8000)
running /root/env/bin/uwsgi
[uWSGI] getting INI configuration from uwsgi.ini
*** Starting uWSGI 2.0.19.1 (64bit) on [Tue Apr 20 01:30:02 2021] ***
compiled with version: 8.3.1 20191121 (Red Hat 8.3.1-5) on 09 April 2021 13:31:58
os: Linux-4.18.0-240.10.1.el8_3.x86_64 #1 SMP Mon Jan 18 17:05:51 UTC 2021
nodename: buaa-2033
machine: x86_64
clock source: unix
detected number of CPU cores: 2
current working directory: /root/ReedSailing-BackEnd/django_backend
detected binary path: /root/env/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to ./
your processes number limit is 30951
your memory page size is 4096 bytes
detected max file descriptor number: 65535
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.8.1 (default, Apr  9 2021, 01:45:33)  [GCC 8.3.1 20191121 (Red Hat 8.3.1-5)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x232e280
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
The -s/--socket option is missing and stdin is not a socket.
*** Starting uWSGI 2.0.19.1 (64bit) on [Tue Apr 20 01:34:21 2021] ***
compiled with version: 8.3.1 20191121 (Red Hat 8.3.1-5) on 08 April 2021 18:25:32
os: Linux-4.18.0-240.10.1.el8_3.x86_64 #1 SMP Mon Jan 18 17:05:51 UTC 2021
nodename: buaa-2033
machine: x86_64
clock source: unix
detected number of CPU cores: 2
current working directory: /root/ReedSailing-BackEnd/django_backend
detected binary path: /root/python/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to ./
your processes number limit is 30951
your memory page size is 4096 bytes
detected max file descriptor number: 65535
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address ./buaa.sock fd 6
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.8.1 (default, Apr  9 2021, 01:45:33)  [GCC 8.3.1 20191121 (Red Hat 8.3.1-5)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x10854c0
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 802120 bytes (783 KB) for 10 cores
*** Operational MODE: preforking ***
added /root/env/lib/python3.8/site-packages/ to pythonpath.
ModuleNotFoundError: No module named 'buaa'
unable to load app 0 (mountpoint='') (callable not found or import error)
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x10854c0 pid: 440173 (default app)
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 440173)
spawned uWSGI worker 1 (pid: 440175, cores: 1)
spawned uWSGI worker 2 (pid: 440176, cores: 1)
spawned uWSGI worker 3 (pid: 440177, cores: 1)
spawned uWSGI worker 4 (pid: 440178, cores: 1)
spawned uWSGI worker 5 (pid: 440179, cores: 1)
spawned uWSGI worker 6 (pid: 440180, cores: 1)
spawned uWSGI worker 7 (pid: 440181, cores: 1)
spawned uWSGI worker 8 (pid: 440182, cores: 1)
spawned uWSGI worker 9 (pid: 440183, cores: 1)
spawned uWSGI worker 10 (pid: 440184, cores: 1)
*** Starting uWSGI 2.0.19.1 (64bit) on [Tue Apr 20 01:36:23 2021] ***
compiled with version: 8.3.1 20191121 (Red Hat 8.3.1-5) on 08 April 2021 18:25:32
os: Linux-4.18.0-240.10.1.el8_3.x86_64 #1 SMP Mon Jan 18 17:05:51 UTC 2021
nodename: buaa-2033
machine: x86_64
clock source: unix
detected number of CPU cores: 2
current working directory: /root/ReedSailing-BackEnd/django_backend
detected binary path: /root/python/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to ./
your processes number limit is 30951
your memory page size is 4096 bytes
detected max file descriptor number: 65535
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address :8000 fd 6
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.8.1 (default, Apr  9 2021, 01:45:33)  [GCC 8.3.1 20191121 (Red Hat 8.3.1-5)]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x1a754c0
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 802120 bytes (783 KB) for 10 cores
*** Operational MODE: preforking ***
added /root/env/lib/python3.8/site-packages/ to pythonpath.
ModuleNotFoundError: No module named 'buaa'
unable to load app 0 (mountpoint='') (callable not found or import error)
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x1a754c0 pid: 440208 (default app)
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 440208)
spawned uWSGI worker 1 (pid: 440210, cores: 1)
spawned uWSGI worker 2 (pid: 440211, cores: 1)
spawned uWSGI worker 3 (pid: 440212, cores: 1)
spawned uWSGI worker 4 (pid: 440213, cores: 1)
spawned uWSGI worker 5 (pid: 440214, cores: 1)
spawned uWSGI worker 6 (pid: 440215, cores: 1)
spawned uWSGI worker 7 (pid: 440216, cores: 1)
spawned uWSGI worker 8 (pid: 440217, cores: 1)
spawned uWSGI worker 9 (pid: 440218, cores: 1)
spawned uWSGI worker 10 (pid: 440219, cores: 1)
